src/crypto/aes.cpp

```diff
@@ -30,6 +30,7 @@ struct aes_encoder::impl
 aes_encoder::aes_encoder()
 {
   static int init = init_openssl();
+  (void)init;
```

Comment:
Is this to suppress an "unused variable" warning?

Reply:
I guess so. Have seen that often in pure C projects that have tight compile settings.